job/server/src/main/java/alluxio/worker/job/task/TaskExecutorManager.java

```diff
@@ -136,6 +138,11 @@ public synchronized void executeTask(long jobId, int taskId, JobConfig jobConfig
     taskInfo.setJobId(jobId);
     taskInfo.setTaskId(taskId);
     taskInfo.setStatus(Status.RUNNING);
+    try {
+      taskInfo.setWorkerHost(InetAddress.getLocalHost().getHostName());
+    } catch (UnknownHostException e) {
+      // ignore
```

Comment:
Why can we ignore this?

Reply:
This is only to stamp the worker host information so the worst case of this is to not have hostname information. But maybe, what I should do instead is to stamp them again from the master.